library/swift/test/EnvoyClientBuilderTests.swift

```diff
@@ -1,4 +1,5 @@
 @testable import Envoy
+import EnvoyEngine
```

Comment:
Out of curiosity, any reason not to use `@_implementationOnly` here for consistency?

Reply:
It doesn't really matter for tests since no one consumes their output swiftmodule, and the annoyance is that if you do it once in a single target you have to do it for all imports, so then everyone has to know about it.